spec/features/user_views_lists_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.